modules/3d-tiles/src/tileset/tileset-3d.js

```diff
@@ -447,31 +489,44 @@ export default class Tileset3D {
   }
 
   async _loadTile(tile) {
-    const expired = tile.contentExpired;
-    const requested = await tile.requestContent(this.DracoLoader);
-
-    if (!requested) {
-      return;
-    }
+    // const expired = tile.contentExpired;
```

Comment:
why this is comment out?

Reply:
As far as I can tell this is not working and I am not sure how it should work. 

We still have a bunch of commented-out, not yet ported code in these files so I didn't want to remove any code just yet.